src/components/Characters/GenderFlower.js

```diff
@@ -1,35 +1,55 @@
-      />
-    </svg>
-  );
-}
-
-const GenderFlower = styled(GenderFlowerArtwork)`
+const GenderFlowerStyled = styled(animated.span)`
```

Comment:
I would call it just _Wrapper_.

Reply:
I will refer to your comments here:

>I would call it just Wrapper.

Fixed in 3af19aae897caccb437ea724e1114c1cbf1368e2

>Personally maybe I would try to overshoot scale a bit above 1 at some point so it will be more bouncy, but rotation is really nice touch.

I've created custom animation based on keyframes using interpolation... but it turned out, that using predefined `config.wobbly` together with *rotate* and *scale*  is  **more bouncy**  than my custom approach based on keyframes - check new GIF image on the top of the PR description

>I think useTransition is more suitable here than useSpring.
After @mgolawski suggestions I've replaced `useSpring` with `useTransition` hook. More clean and elegant solution. And of course now animation works when component is mounted and unmounted.

Updated in 3af19aae897caccb437ea724e1114c1cbf1368e2

Reply:
Yeah, overall prefer spring / config driven animations than interpolation based ones. The look more smooth and natural. Interpolations are only good if you need full control and to achieve very specific result.